data_steward/tools/create_tier.py

```diff
@@ -0,0 +1,143 @@
+# Python imports
+import argparse
+import logging
+import re
+
+# Third party imports
+
+# Project imports
+from utils import pipeline_logging
+
+LOGGER = logging.getLogger(__name__)
+
+TIER_LIST = ['controlled', 'registered']
+DEID_STAGE_LIST = ['deid', 'base', 'clean']
+
+
+def validate_tier_param(tier):
+    """
+    helper function to validate the tier parameter passed is either 'controlled' or 'registered'
+
+    :param tier: tier parameter passed through from either a list or command line argument
+    :return: nothing, breaks if not valid
+    """
+    if tier.lower() not in TIER_LIST:
+        LOGGER.error(
+            f"Parameter ERROR: {tier} is an incorrect input for the tier parameter, accepted: controlled or "
+            f"registered")
+
+
+def validate_deid_stage_param(deid_stage):
+    """
+    helper function to validate the deid_stage parameter passed is correct, must be 'deid', 'base' or 'clean'
+
+    :param deid_stage: deid_stage parameter passed through from either a list or command line argument
+    :return: nothing, breaks if not valid
+    """
+    if deid_stage not in DEID_STAGE_LIST:
+        LOGGER.error(
+            f"Parameter ERROR: {deid_stage} is an incorrect input for the deid_stage parameter, accepted: "
+            f"deid, base, clean")
```

Comment:
Logging is nice.  Raising an error is even better.  We can't know which data_stage tier we are working with if this isn't provided.  Go ahead and raise an error.

Reply:
I've added in `raise argparse.ArgumentTypeError` I did leave in the `LOGGER.error` let me know if that is unnecessary and I should remove?

Reply:
I will not ask to remove logging statements!